.github/workflows/scripts/aot-demo.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -ex
+
+export TI_SKIP_VERSION_CHECK=ON
+export TI_CI=1
+
+. $(dirname $0)/libcommon.sh
+
+
+function build-and-smoke-test-android-aot-demo {
+    pushd taichi
+    GIT_COMMIT=$(git rev-parse HEAD | cut -c1-7)
+    popd
+
+    export TAICHI_REPO_DIR=$(pwd)/taichi
+
+    git clone https://github.com/taichi-dev/taichi-aot-demo
+    APP_ROOT=taichi-aot-demo/implicit_fem
+    ANDROID_APP_ROOT=$APP_ROOT/android
+    JNI_PATH=$ANDROID_APP_ROOT/app/src/main/jniLibs/arm64-v8a/
+
+    # cd $APP_ROOT/python
+    # python3 implicit_fem.py --aot
+
+    mkdir -p $JNI_PATH
+    cp taichi/build/libtaichi_export_core.so $JNI_PATH
+    cd $ANDROID_APP_ROOT
+    sed -i "s/TaichiAOT/AOT-$GIT_COMMIT/g" app/src/main/res/values/strings.xml
+    ./gradlew build
+
+    run-android-app \
+        app/build/outputs/apk/debug/app-debug.apk \
+        com.taichigraphics.aot_demos.implicit_fem/android.app.NativeActivity
+}
+
+function prepare-unity-build-env {
+    cd taichi
+
+    # Dependencies
+    git clone --reference-if-able /var/lib/git-cache https://github.com/taichi-dev/Taichi-UnityExample
```

Comment:
do we really need `reference-if-able` here? it generates warnings...

Reply:
The warning is benign.
This is not strictly needed, but it will speed up clone if the cache is usable.

Reply:
ehmm. i expect a clean git-cache for every new container...

Reply:
It's bind mounted to host :)